<launch>

    <node pkg="computer_vision" type="activation_node.py" name="ActivationTest" output="screen"/>
    
    <node pkg="computer_vision" type="tracker_live_node.py" name="VideoMotionTracker" output="screen" />

    <node pkg="computer_vision" type="tracker_video_node.py" name="LiveMotionTracker" output="screen" />

    <node pkg="computer_vision" type="ur_node.py" name="UR" output="screen" />

    <node pkg="computer_vision" type="project_setup_node.py" name="ProjectSetup" output="screen" />
    
</launch>